app/views/admin/kudos/edit.html.erb

```diff
@@ -0,0 +1,3 @@
+<h1>Editing Admin Kudo</h1>
```

Comment:
I would consider renaming it to simply 'Editing Kudo', since you are working with `Kudo` Model

Reply:
You are right, I thought about this. Will fix.